### PR TITLE
Production hardening

### PR DESCRIPTION
**Pull request by eduartua**

### CODE REVIEW

File: terraform/dev/main.tf [hunk 1]
```diff
@@ -17,7 +17,7 @@ terraform {
 }
 
 module "gke" {
-  source = "git::git@github.com:cloudkite-io/terraform-modules.git//modules/gke"
+  source = "git::git@github.com:cloudkite-io/terraform-modules.git//modules/gke?ref=v0.0.1"
```
Comment (victortrac) [inline terraform/dev/main.tf:20]: URL has an extra `/`.

Comment (eduartua) [reply terraform/dev/main.tf:20]: It requires the extra `/`

Comment (eduartua) [reply terraform/dev/main.tf:20]: https://www.terraform.io/docs/modules/sources.html#modules-in-package-sub-directories

File: terraform/dev/config.tfvars [hunk 2]
```diff
@@ -4,4 +4,23 @@ gcp = {
...
+  min_master_version = "1.14.3-gke.11"
+  master_ipv4_cidr_block = "172.31.0.0/28"
+}
+
+gke_nodepools = [
+  {
+    auto_repair = true
+    auto_upgrade = false
+    min_node_count = 1
+    max_node_count = 100
```
Comment (victortrac) [inline terraform/dev/config.tfvars]: This is dev, so probably 10 is reasonable.

**Review by victortrac — APPROVED** [review]

**Pull request merged**